packages/services/core-analytics/src/services/analytics.mapper.js

```diff
@@ -0,0 +1,137 @@
+import jwtDecode from 'jwt-decode';
+import SHA256 from 'crypto-js/sha256';
+import Cookies from 'universal-cookie';
+import {
+    COUNTRY_INFO,
+    DEFAULT_APP_TYPE,
+    DEFAULT_APP_ID,
+    IDENTITY_PROVIDERS,
+    GRANT_TYPES
+} from '../constants';
+
+/**
+ * Returns the current orientation name.
+ *
+ * @return {string} The orientation name
+ */
+const getOrientation = () => {
+    if (typeof (window) !== 'undefined') {
+        const height = window.innerHeight;
+        const width = window.innerWidth;
+
+        return height > width ? 'Portrait' : 'Landscape';
+    }
+
+    return undefined;
+};
+
+/**
+ * Returns the contents of a cookie
+ *
+ * @param {string} name - The name of the cookie
+ * @return {string} The contents of the cookie
+ */
+const getCookie = cookieName => {
+    const cookies = new Cookies();
+    const value = cookies.get(cookieName);
+
+    return value;
+};
+
+/**
+ * Maps the anonymous User Id to the UserData 'a-UserId' field.
+ *
+ * @return {string} UserId from je-auser cookie
+ */
+const mapAnonymousUserId = () => {
+    const value = getCookie('je-auser');
```

Comment:
Perhaps drive this cookie name from config (maybe next pr?) as I think we call it `jet-auser` in certain codebases